src/baguetteBox.js

```diff
@@ -316,6 +325,8 @@
             options.animation === 'slideIn' ? '' : 'none');
         // Hide buttons if necessary
         if (options.buttons === 'auto' && ('ontouchstart' in window || imagesMap[currentGallery].length === 1)) {
+            previousButtonContent.setAttribute('aria-hidden', 'true');
+            nextButtonContent.setAttribute('aria-hidden', 'true');
             options.buttons = false;
         }
         // Set buttons style to hide or display them
```

Comment:
Okay, you're right. The last patch doesn't look so good. We could move to `aria-hidden=true` approach, but what about the Close button? It's always visible, so does it make sense to switch `aria-hidden` for him?